modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-consume.adoc

```diff
@@ -0,0 +1,92 @@
+= Anypoint MQ Consume Operation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The `consume` operation in the Anypoint MQ connector provides the ability to consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+
+== Incoming Message Structure
+
+Every consumed message consists of two parts:
+
+* `payload` - Represents the value of the received message body
+* `attributes` - Contains metadata regarding the received message, such as the message properties, or its ID.
+
+image::anypoint-mq/3.x/amq-3x-consume-attributes.png[Anypoint MQ Message Attributes]
+
+
+== Consume a Message
+
+The simplest declaration to consume a message from a queue is to drag the *Consume* operation from the Mule palette to the canvas and configure the `Queue` parameter with the name of the queue from which to retrieve the message:
+
+image::anypoint-mq/3.x/amq-3x-consume-operation.png[Anypoint MQ Consume Operation]
+
+For example:
+
+[source,xml,linenums]
+----
+<anypoint-mq:consume config-ref="Anypoint_MQ_config" destination="myQueue"/>
+----
+
+The Consume operation in this example consumes the first available message in `myQueue` and converts it to a `MuleMessage` with this structure:
+
+* The message's body as payload.
+* The message's metadata in the message attributes. Attributes might include the `messageId` or the `redeliveryCount`.
+
+TIP: Use a DataWeave expression in the `Queue` parameter to dynamically change the target queue from which to consume the message.
```

Comment:
Can this be plain text instead of a Tip?
The theory behind limiting, if not altogether omitting, notice formats is that everything we document is necessary, without fluff and without relative weight.